src/cascadia/TerminalConnection/AzureConnection.cpp

```diff
@@ -0,0 +1,478 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#include "pch.h"
+#include "AzureConnection.h"
+#include "AzureClientID.h"
+#include <sstream>
+#include <stdlib.h>
+
+// STARTF_USESTDHANDLES is only defined in WINAPI_PARTITION_DESKTOP

```

Comment:
Wait, what? Why is this a problem. We should be in WINAPI_PARTITION_DESKTOP... aren't we?

Reply:
This is also there in ConhostConnection.cpp (which I was using as a reference). Will remove it from both files.